.travis.yml

```diff
@@ -43,8 +44,8 @@ script:
   - ./tests/idempotence_check.sh idempotence_out
 
   # Testing with docker (experimental)
```

Comment:
If this is experimental @WolfgangPecho, then should we remove it and put it into a separate PR?